edx_proctoring/admin.py

```diff
@@ -104,7 +104,7 @@ def queryset(self, request, queryset):
 
         if self.value() == 'all_unreviewed':
             return queryset.filter(reviewed_by__isnull=True)
-        elif self.value() == 'all_unreviewed_failures':
+        if self.value() == 'all_unreviewed_failures':
```

Comment:
Whats the purpose of this change.

Reply:
According to pylint if you shouldn't use else after a return in if block.

https://pheanex.github.io/pylint/